scripts/TVs/add_new_media.py

```diff
@@ -94,6 +94,11 @@ def add_new_media(username=None, tv=None):
             username_invalid = False
 
         tv = guess_tv(username)
+        if tv is 'q':
+            # restart while loop
+            media_url = True
+            username_invalid = True
+            return
```

Comment:
This doesn't restart the while loop, `return` goes back to the main menu.  Did you test it?

Reply:
my bad I forgot syntax, fixed in [9d5ba82](https://github.com/timberline-secondary/hackerspace-scripts-2/pull/131/commits/9d5ba82911016209ffb31732ae226ba37b4995f1)